lib/builder.js

```diff
@@ -264,7 +264,7 @@ function executeConfigFile(saveForReset, ignoreBaseURL, source) {
   };
 
   // jshint evil:true
-  new Function(source.toString()).call(global);
+  eval(source.toString());
```

Comment:
I did some tests, I'm seeing that for both `eval(...)` and new `Function(...)` , assignment to undeclared variable causes creation of that variable in global scope. The only sure way to prevent that is to `use strict`, that is

`eval('(function() {"use strict"; ' + source.toString() + '\n})()')`

It would not prevent from assigning to global variables that already exist in global scope, however - I don't think there is any protection from that.

Now I'm wondering how many people have config files that will break in strict mode.

Thanks for the hint, I will update the PR with `eval function using strict`.